src/wallet/rpcwallet.cpp

```diff
@@ -3287,6 +3289,14 @@ UniValue rescanblockchain(const JSONRPCRequest& request)
     return response;
 }
 
+// Deprecated account RPCs.
+UniValue getaccountaddress(const JSONRPCRequest& request) { return getlabeladdress(request); }
+UniValue getaccount(const JSONRPCRequest& request) { return getlabel(request); }
+UniValue getaddressesbyaccount(const JSONRPCRequest& request) { return getaddressesbylabel(request); }
+UniValue getreceivedbyaccount(const JSONRPCRequest& request) { return getreceivedbylabel(request); }
+UniValue listreceivedbyaccount(const JSONRPCRequest& request) { return listreceivedbylabel(request); }
+UniValue setaccount(const JSONRPCRequest& request) { return setlabel(request); }
```

Comment:
This allows for no way for the method to know whether they've been called as labelXX or accountXX, as we might want to implement slightly different behavior for both, that seems important information?